### ISSUE TITLE
Export fails: Building accelerated plugins...

### ISSUE THREAD

**Issue by JimKlingshirn** [NVIDIA/retinanet-examples#22]
Thanks for the excellent code...

I've successfully trained retinanet on coco. The accuracy looks decent. Then, I converted the retinanet_rn50fpn.pth model to .onnx.  Then I built the export and infer utils successfully.  So far, so good.  But when I try to export the .onnx model to .path, it fails a Parameter check.
```
./export retinanet_rn50fpn.onnx out.path

...... hundreds of lines of output.. and then

 530:Conv -> (256, 80, 80)
Building accelerated plugins...
Parameter check failed at: ../builder/Network.cpp::addConcatenation::167, condition: nbInputs > 0 && nbInputs < MAX_CONCAT_INPUTS
Segmentation fault (core dumped)
```
I'm using TensorRT 5.1.2.2  on a Ubuntu 16.04 desktop, with Cuda Driver Version: 418.56       CUDA Version: 10.1 

Thanks

**Comment by JimKlingshirn** [comment]
Okay, I see this is a duplicate issue....  Apparently I need to specify --opset=9 when I create the onnx file.  When I do so, it works.

**Comment by jinfagang** [comment]
@JimKlingshirn JimKlingshirn  What do u mean specify --opset?? It already using opset9 in 

```
        import torch.onnx.symbolic_opset9 as onnx_symbolic

```

**Comment by JimKlingshirn** [comment]
I specified the opset when calling torch.onnx.export, using extra_args.  For example:
```
    extra_args = { 'opset_version': 9 }  
    torch.onnx.export(net.cuda(), zero_input, onnx_bytes, verbose, input_names=["data"] ,*extra_args)
```

**Comment by jinfagang** [comment]
@JimKlingshirn  I don't get your point, this repo doesn't using `torch.onnx.export` export onnx model at all, it using Engine to parse it, which line of code does `torch.onnx.export` used? Did u using the 19.09 branch or old branch?
